common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -149,6 +149,18 @@ class InheritanceMixin(XBlockMixin):
         default=True,
         scope=Scope.settings
     )
+    video_bumper = Dict(
+        display_name=_("Video Bumper"),
+        help=_(
+            """Identify a video, 5-15 seconds in length, to play before course videos. Enter the video ID from"""
+            """ the Video Uploads page and zero or more transcript files in the following format:"""
```

Comment:
Please change "zero" to "one" here. (Because we want to encourage teams to make everything they include in the course accessible, there really should be at least one transcript file.)